### DISCUSSION TITLE
Add Ability to Sell Sub-Wide to Multi Player

### DISCUSSION THREAD

**Discussion by gerakis** [FakeFishGames/Barotrauma#7757] [Ideas and suggestions]
![i cant sell all this by hand](https://user-images.githubusercontent.com/16128947/145897569-f2adb583-ac94-4d6c-b2d0-9dff062b1cdb.png)

Are you kidding me with this? Why is the sub-wide sales tab not in multiplayer? This is nuts. I cant be expected to cart this all up to the shop one crate at a time. I'm going to get carpal tunnel syndrome just clicking all the buds. This is 25 trips back and forth up the ladder. This is close to 6000 mouse clicks. Are the devs sadistic? Why would they do this to us?

Its already programmed into the game, just copy it from singleplayer to multiplayer. This works just fine in singleplayer.

**Comment by WickedWolfy** [comment]
Enabling a dedicated "For Sale" locker on the sub with a specific tag would be a grand alternative.
Otherwise +1

**Comment by ekqvist** [comment]
Hi! We hope to also bring the feature to multiplayer. However, the multiplayer implementation does require some extra work which is why it wasn't implemented at the same time as the singleplayer feature. But as I said, we hope to bring the feature to the multiplayer game as well when we can 🙂
